test/rocc/README

```diff
@@ -0,0 +1,28 @@
+This tests basic RoCC accelerator functionality using the default dummy_rocc
+accelerator included with spike. You may use this as a template for writing
+bare-metal software for RoCC accelerators.
+
+# Key Files
+* crt.S: Initializes the system and sets up the basic runtime. This eliminates the need for an OS.
+* rocc.h: These are helper macros for constructing RoCC instructions
+* syscalls.c: This implements a subset of the C standard library. We do not
+  include all of C, so you'll need to implement whatever you need that isn't
+  there.
+* link.ld: This instructs the linker to setup the binary as a standalone
+  bootable image on the bare-metal hardware without any OS.
+* rocc-test.c: This is our actual benchmark. For the most part, a new project
+  would only need to modify this file.
+
+# Usage
+Build with:
+    make
+
+Run on Spike with:
+    spike --extension=dummy_rocc rocc-test.riscv
+
+Run in Marshal:
+    ./marshal build test/rocc.json
+    ./marshal launch -s test/rocc.json
```

Comment:
"Run in Marshal" here is "Run in FireSim"?
(Assuming we're moving marshal up on level in the hierarchy, this is not trivial anymore. Need to differentiate between running in FireSim, running in Verilator, or running in QEMU/Spike)

Reply:
No, this is "Run using Marshal" (using spike). I don't include instructions on running firesim in this readme, I suppose I could.

Reply:
Then maybe make this "Run in Marshal using Spike"?
(I think it would be useful to have instructions for bare-metal on FireSim and Verilator. We have the Explicit Verilator instructions in the software-simulation section, but then it's not clear how it relates to Marshal)